src/attributes.ts

```diff
@@ -141,8 +145,7 @@ attributes['style'] = applyStyle;
 /**
  * Calls the appropriate attribute mutator for this attribute.
  */
-function updateAttribute(
-    el: HTMLElement, name: string, value: {}|null|undefined) {
+function updateAttribute(el: Element, name: string, value: {}|null|undefined) {
```

Comment:
use unknown instead of {}? since this could be a string or number instead of an object.

Reply:
same above

Reply:
Changed to `unknown`